templates/bundles/TwigBundle/Exception/error.html.twig

```diff
@@ -0,0 +1,26 @@
+{% extends '@W3CWebsiteTemplates/base.html.twig' %}
+
+{% block main %}
+    <h1>{% block title %}{{ page.title|default('Error: {{ status_code }} - {{ status_text }}') }} | W3C{% endblock %}</h1>
+    <p>{{ 'error.intro'|trans({'status_code': status_code, 'status_text': status_text}, 'w3c_website_templates_bundle') }}</p>
+
+    {% block stacktrace %}
+        {% if is_granted('ROLE_ADMIN') and exception %}
+            <h2>Error details</h2>
+            <p>Message: <strong>{{ exception.message }}</strong></p>
+            <p>in <em>{{ exception.file }}: {{ exception.line }}</em></p>
+            <p>Trace:</p>
+            <pre>{{ exception.traceAsString }}</pre>
```

Comment:
Use a component:
```suggestion
            <div class="component component--text">
                <h2>Error details</h2>
                <p>Message: <strong>{{ exception.message }}</strong></p>
                <p>in <em>{{ exception.file }}: {{ exception.line }}</em></p>
                <p>Trace:</p>
                <pre>{{ exception.traceAsString }}</pre>
            </div>
```

Reply:
Integrated in https://github.com/w3c/w3c-website-templates-bundle/commit/776d66e897902c8c85388521c3a2561fa634b319